README.md

```diff
@@ -2,11 +2,22 @@
 > Application to automate the selection of meals and creation of shopping lists.
 
 ## Dev
-
+1. Create an `.env-dev` file. It can be empty.
+2. Startup dev environment
 ```
 docker-compose -f docker-compose-dev.yml up
 ```
 
+### Using `pdb` with
+```
```

Comment:
add a comment about the entry point being empty instead of `python manage.py runserver`